OneCommunityGlobal/HighestGoodNetworkApp
# Roberto- fix adding person to existing team

# Description
While creating a new user, assigning them to a team didn't register on Teams section.
<img width="835" alt="Screenshot 2023-09-23 at 5 06 28 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/117053202/ba82f413-ee14-41d8-b4b0-1652aa747485">

Fixes # (bug list priority high/medium/low x.y.z)
Or Implements # (WBS) 

## Related PRS (if any):
…

## Main changes explained:
- Added a function in UserProfileAdd file that assigns user to teams After the user is created. In order to add a member to a team userId is needed first so function executes as soon as the modal returns a userId.
- Edited the TeamsTab file to perform the same function.
…

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as admin/owner user
4. go to `dashboard→ Other Links→ User Management→ Create New User`
5. Fill out the modal and assign the new user to a team. Now go to `Other Links→ Teams`, look up the team and verify the user you created is showing in the member list. 
6. Additionally visit that user's account → Teams Tab → Assign them to another team and verify that it works as well.

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/117053202/ec59d0aa-af02-4aea-985d-d8230c3b4928


## Note:
You might see this in the console but not related:
<img width="835" alt="Screenshot 2023-09-23 at 5 23 49 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/117053202/f3a91de1-7742-456a-86ef-8c6bfa0fd356">
